certora/specs/Delegation.spec

```diff
@@ -6,17 +6,18 @@ methods {
     function delegationNonce(address) external returns uint256 envfree;
 }
 
-// Ghost variable to hold the sum of voting power.
-// To reason exhaustively on the value of of delegated voting power we proceed to compute the partial sum of balances for each possible address.
+// To reason exhaustively on the value of of delegated voting power we proceed to compute the partial sum of delegated votes to parametre A for each possible address.
 // We call the partial sum of votes to parameter A up to an addrress a, to sum of delegated votes to parameter A for all addresses within the range [0..a[.
```

Comment:
those comments are about `sumsOfVotesDelegatedToA` which I think would be better to put before the definition of `sumsOfVotesDelegatedToA`. Instead here we could put a similar paragraph for the new `sumOfVotes`

Reply:
Done in https://github.com/morpho-org/morpho-token/pull/104/commits/9a935dd44f6503428181604b55ee6074f46864d0.